src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/Maintainability/ReviewUnusedParametersTests.Fixer.cs

```diff
@@ -511,6 +511,25 @@ class MyValue
             await VerifyCS.VerifyCodeFixAsync(code, fix);
         }
 
+        [Fact]
+        public async Task Operators_CSharp()
+        {
+            var code = @"
+public struct Unit
+{
+    public static bool operator ==(Unit [|x|], Unit [|y|]) => true;
+    public static bool operator !=(Unit [|x|], Unit [|y|]) => false;
+}";
+
+            var fixedCode = @"
+public struct Unit
+{
+    public static bool operator ==(Unit _, Unit _1) => true;
+    public static bool operator !=(Unit _, Unit _1) => false;
+}";
```

Comment:
The fix all is failing for this, it produces `_` for all parameters.
Can someone give me a guidance on how to fix this?